command/daemon/config.go

```diff
@@ -49,7 +49,7 @@ type Config struct {
 		EnvFile     string            `envconfig:"DRONE_RUNNER_ENV_FILE"`
 		Secrets     map[string]string `envconfig:"DRONE_RUNNER_SECRETS"`
 		Labels      map[string]string `envconfig:"DRONE_RUNNER_LABELS"`
-		Volumes     map[string]string `envconfig:"DRONE_RUNNER_VOLUMES"`
+		Volumes     []string          `envconfig:"DRONE_RUNNER_VOLUMES"`
```

Comment:
Could we use `CI_MOUNT_VOLUMES` similar to Harness documentation and keep the older env variable in the legacy map?